x-pack/filebeat/input/azureeventhub/eph.go

```diff
@@ -74,3 +86,15 @@ func (a *azureInput) runWithEPH() error {
 	}
 	return nil
 }
+
+func getAzureEnvironment(overrideResManager string) (azure.Environment, error) {
+	// if no overrride is set then the azure public cloud is used
+	if overrideResManager == "" {
+		return azure.PublicCloud, nil
+	}
+	if env, ok := environments[overrideResManager]; ok {
+		return env, nil
+	}
```

Comment:
Nit. Should we try without the `/` paht? So it works both for `https://management.azure.com/` and `https://management.azure.com`.